docs/external-interop/origin_avalanche/summary.md

```diff
@@ -0,0 +1,52 @@
+section.
+
+The primary difference between internal and external interoperability is how message authentication is accomplished. For
+internal interoperability, this was achieved used a pre-compiled smart contract called `Warp`. An alternative solution
+is necessary for external EVM chains.
+
+This will also require changing the relayers. They will need clients for external EVM chains and will need to call
+different contract functions and provide different data as well. There also new duties required of relayers other than
+just sending messages that will be required. This is further describe in the [Relayer changes](relayer_changes.md) section.
+
+## Root of trust
+
+The P-chain of Avalanche is root of trust for all aspects of the protocol when authenticating messages from Avalanche or
+an Avalanche L1. Specifically, all authentication proofs should derive their authority from a specific initial 
+P-chain validator set known to external EVM chain. While the publication of this initial set may be done in a centralized
+manner, it's validity is publicly auditable and thus a trustless procedure. 
+
+As P-chain validator sets change over time, a __chain of custody__ must be maintained whereby a quorum the current P-chain 
+validator set must sign off on the next validator set (_N.B. we will often say that validator set signs a message, but 
+this should always be interpreted as meaning a quorum of signatures_). This new set will be published on the external EVM 
+chain if the signature check passes. From then on, this will be the current P-chain validator set whose signatures represent 
+authority of Avalanche. 
+
+For more details on how P-chain validator sets are published on external EVM chains, see the [Avalanche validator set registry](validator_set_registry.md) 
+section. As for how validator sets changes are communicated, see the [Updating validator sets]() section.
+
+## Avalanche L1s
+
+Communication to external EVM chains is designed to be possible from Avalanche L1s, as well the Avalanche P-chain. To
+achieve this, we extend the power to authorize messages from L1s to the validator set of said L1. Much like the P-chain,
+a current validator set of an L1 is published to the external EVM chain. Messages from an L1 should be signed by a quorum
+of this validator set in order to be authenticated. 
+
+However, the protocol differs when the very first set of validators for an L1 is published to the external EVM chain. This
+initial set must be signed by the current P-chain validator set that is published on the external EVM chain. After an L1
+is registered to the external EVM chain in this way, it may update itself via a quorum of signatures of the currently 
+published L1 validator set, forming a chain of custody back to the originally registered set. Again, for more details,
+see the [Avalanche validator set registry]() section.
```

Comment:
It would also probably be helpful if an L1 is able to update itself with a message signed by _either_ a quorum of it's own validators or a quorum of p-chain validators - in case the L1 has not updated it's validator set in a while and can't construct a valid message.